src-web/components/modals/PolicyActionModal.js

```diff
@@ -6,8 +6,9 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
-import { Modal, Notification } from 'carbon-components-react'
-import { Spinner } from '@patternfly/react-core'
+import '@patternfly/react-core/dist/styles/base.css'
```

Comment:
in theory, we shouldn't need to import the base css in each file anymore as we already imported the whole patternfly css in scss/common.scss